lib/reline/config.rb

```diff
@@ -219,6 +224,7 @@ def read_lines(lines, file = nil)
     unless if_stack.empty?
       raise InvalidInputrc, "#{file}:#{if_stack.last[0]}: unclosed if"
     end
+    @loaded = true
```

Comment:
For readability, I'd prefer this to be set in `config.read`, maybe after line 148. WDYT?

Reply:
Yes, either `read` L148 or `read_lines` would work. `read_lines` is a public API, so I thought it would be better to set it here.

Reply:
Technically `read_lines` is public as it's not declared private, but functionally do we need/want to expose it in addition to `read`? I feel it's more like a implementation detail that should be hide under `read`. And currently it doesn't have other callers either.

Reply:
If we want to load something other than a file as `.inputrc`, using `read_lines` could be an option.
However, in real applications, it's unlikely that `.inputrc` would be anything other than a file.
Functionally, it seems unnecessary to keep `read_lines` as a public API. It might have been made public primarily for testing purposes.

https://github.com/ruby/reline/blob/3f27286a5ec83f8aa4cd47217768499b26fcbc1a/test/reline/test_config.rb#L26-L28

I'll set it up using `read`. 👍